packages/components/inputs/creatable-select-input/src/creatable-select-input.stories.tsx

```diff
@@ -0,0 +1,164 @@
+      { value: 'lion', label: 'Lion' },
+      { value: 'monkey', label: 'Monkey' },
+      { value: 'kangaroo', label: 'Kangaroo' },
+    ],
+  },
+  {
+    options: [
+      { value: 'flamingo', label: 'Flamingo' },
+      { value: 'moose', label: 'Moose' },
+      { value: 'prairie-dog', label: 'Prairie Dog', isDisabled: true },
+      { value: 'snake', label: 'Snake' },
+      { value: 'porcupine', label: 'Porcupine' },
+      { value: 'stingray', label: 'Stingray' },
+      { value: 'panther', label: 'Panther' },
+      { value: 'lizard', label: 'Lizard' },
+      { value: 'parrot', label: 'Parrot' },
+      { value: 'dolphin', label: 'Dolphin' },
+      { value: 'chicken', label: 'Chicken' },
+      { value: 'sloth', label: 'Sloth' },
+      { value: 'shark', label: 'Shark' },
+      { value: 'ape', label: 'Ape' },
+      { value: 'bear', label: 'Bear' },
+      { value: 'cheetah', label: 'Cheetah' },
+      { value: 'camel', label: 'Camel' },
+      { value: 'beaver', label: 'Beaver' },
+      { value: 'armadillo', label: 'Armadillo' },
+    ],
+  },
+  {
+    options: [
+      { value: 'tiger', label: 'Tiger' },
+      { value: 'llama', label: 'Llama' },
+      { value: 'seal', label: 'Seal' },
+      { value: 'hawk', label: 'Hawk' },
+      { value: 'wolf', label: 'Wolf' },
+      { value: 'yak', label: 'Yak' },
+      { value: 'rhinoceros', label: 'Rhinoceros' },
+      { value: 'alpaca', label: 'Alpaca' },
+      { value: 'zebra', label: 'Zebra' },
+      { value: 'cat', label: 'Cat' },
+      { value: 'rabbit', label: 'Rabbit' },
+      { value: 'turtle', label: 'Turtle' },
+      { value: 'cow', label: 'Cow' },
+      { value: 'turkey', label: 'Turkey' },
+      { value: 'deer', label: 'Deer' },
+    ],
+  },
+];
+
+type SelectedOption = {
+  value: string;
+  label: string;
+  __isNew__?: boolean;
+};
+
+export const BasicExample: Story = (args) => {
+  const [value, setValue] = useState<
+    SelectedOption | SelectedOption[] | undefined
+  >([]);
+
+  useEffect(() => {
+    setValue(args.isMulti ? [] : undefined);
+  }, [args.isMulti]);
+
+  return (
+    <div style={{ minHeight: 350 }}>
+      <Spacings.Stack scale="l">
+        <CreatableSelectInput
+          value={value}
+          {...args}
+          onChange={(e) => {
+            setValue(e.target.value as SelectedOption | SelectedOption[]);
+          }}
+          /** needs to be set to undefined (cause storybook adds an empty function via args) */
+          onCreateOption={undefined}
+        />
+        <strong>state:</strong>
+        <pre>
+          {JSON.stringify(
+            {
+              value,
+            },
+            null,
+            2
+          )}
+        </pre>
+      </Spacings.Stack>
+    </div>
+  );
+};
+
+BasicExample.args = {
+  horizontalConstraint: 'scale',
+  options,
+  isMulti: false,
+  hasError: false,
+  hasWarning: false,
+  backspaceRemovesValue: true,
+  isClearable: false,
+  isSearchable: true,
```

Comment:
not in scope for this PR, but why even make `isSearchable` a prop if there's no way to create a new option if it's `false`?